src/dbnode/generated/thrift/rpc.thrift

```diff
@@ -300,6 +300,11 @@ struct NodeHealthResult {
 	1: required bool ok
 	2: required string status
 	3: required bool bootstrapped
+	4: optional ServerMetadata serverMetadata
+}
+
+struct ServerMetadata {
+	1: required bool supportsCompressSnappy
```

Comment:
Would this be better as a list of compression types it supports? Should make the rpc more extensible